SayTheirNames/Source/Service/Networking/NetworkRequestor.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  NetworkRequestor.swift
+//  Say Their Names
+//
+//  Created by evilpenguin on 6/3/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+// MARK: - NetworkRequestor
+
+class NetworkRequestor: NSObject, ServiceReferring {
```

Comment:
```suggestion
final class NetworkRequestor {
```
I think it wouldn't need a service, if it just makes requests and parses responses 🤔 the reversed, `Service` holds the requestor. Otherwise there's circular dependencies...

Reply:
Fixed